20160615

MiguelMoll opened the pull request:
Closes #769 - use GNU tar for builds
Closes #764 - Allow a rack to only operate on itself
Closes #767 - AppShow via the controller only returns an app

## Multi-PR Release Playbook
- [x] Create a pre-release branch
- [x] Merge other PRs to include in the release
- [x] Release branch
- [x] Pass CI
- [x] Code review
- [ ] Create release notes and docs PRs
- [ ] Merge into master
- [ ] Release master
- [ ] Pass CI
- [ ] Update dev and testing racks
- [ ] If rack or app formation changes, test upgrade → re-deploy existing app → downgrade cycle
- [ ] Create [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

nzoschke commented:
Looks good to me. I would like to manually test some of the app name related stuff when there is a branch release.

MiguelMoll commented:
## Improve rack operations

This enhancement addresses issues where a rack name could be used in the context of an app name. The fix includes better error handling to let the user know what exactly is going on.

For more information see PR #764 and #767

## Bug Fix

For a small time window, limited set of users experienced an issue where they weren't able to deploy their apps. This fix address the problem by using GNU tar during builds. 

For more information see PR #769 - Use GNU tar for builds
